src/server/start.js

```diff
@@ -1,8 +1,5 @@
 const mongoose = require('mongoose');
 
-// Import variables.env file
```

Comment:
This was originally here so the server recognized where to find the .env file? 

So if this is gone, how does the server know where to pick up the .env file and the variables in it?